### ISSUE TITLE
[FE] Build static landing page in Staging environment

### ISSUE THREAD

**Issue by brianalloyd** [department-of-veterans-affairs/va.gov-team#35161]
## Issue Description
_ORDMI PIFs have never actually produced any design mockups for the harassment tool. They took conversations with business owners, synthesized requirements from these, then directly put them into code using the https://VA.gov CMS, strictly to save time. They will share screenshots the prototype and the business requirements assembled.  We'll need to take these screenshots and requirements and produce envision mockups that are compliant to VSA/VA.gov design systems.  The designs will be used for both design intent review and initial FE build of the prototype_

- Link to [Product Outline](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/harassment-reporting/product-outline.md)
- Link to the tool or feature in [Drupal](http://preview-prod.vfs.va.gov/preview?nodeId=34641)
[How To Report Harassment At VA _ Veterans Affairs.pdf](https://github.com/department-of-veterans-affairs/va.gov-team/files/7196438/How.To.Report.Harassment.At.VA._.Veterans.Affairs.pdf)
-  Link to PDF Version of the tool.
[How To Report Harassment At VA _ Veterans Affairs.pdf](https://github.com/department-of-veterans-affairs/va.gov-team/files/7196436/How.To.Report.Harassment.At.VA._.Veterans.Affairs.pdf)
[content](http://preview-prod.vfs.va.gov/preview?nodeId=34641)
- Link to [Content Landing Page](http://preview-prod.vfs.va.gov/preview?nodeId=34641) 

## Frontend tasks
- Create a static landing page from this [preview](http://preview-prod.vfs.va.gov/preview?nodeId=34641) in "Staging" with required interactions 
- Collaborate with researcher to determine items needed ISO research plan/convo guide
- Validate with @cindymerrill ISO research efforts 
- Update/Finalize as needed/determined

## Acceptance Criteria
- [x] Research comments on this issue to approve staging prototype for review
- [x] Staging Prototype for review by SH/PM/PO (Required)
- [x] Researcher reviews prototype ISO research plan/Convo guide 
- [x] Feedback is documented on ticket for follow up and review

**Comment by brianalloyd** [comment]
@tmac9494 do you have the URL/lInk to support validation of the new page?  Please post here so we review and validate.  Thanks.

**Comment by tmac9494** [comment]
https://github.com/department-of-veterans-affairs/vagov-content/pull/821

**Comment by cindymerrill** [comment]
Reviewed static landing page on Staging left feedback for @tmac9494 in slack. Looking forward to seeing the revisions.

**Comment by cindymerrill** [comment]
There will be changes to the XXX phone numbers on the page, which hopefully will get decided early next week. I'll let @tmac9494 know when I get the new copy.

**Comment by davidconlon** [comment]
@cindymerrill @brianalloyd @tmac9494 I'm really confused right now.
The "Static Page" is simply a Benefits Detail page coming out of Drupal.  

There shouldn't be any FE work to be done. it's all content in Drupal. 

@DanielleThierryUSDSVA is probably the person we need to coordinate with here.

**Comment by cindymerrill** [comment]
@davidconlon I believe that Trent had to re-create the page on Staging in order for me to be able to usability test with screen reader users. And when I reviewed the page, I found a few issues (documented [here](https://dsva.slack.com/archives/C52CL1PKQ/p1643917931308419)), which Trent said he's fixed but I haven't been able to review on Staging yet.

When the text with the phone numbers gets settled with your stakeholders, Trent will have to make the same changes in the version on Staging.

**Comment by brianalloyd** [comment]
@davidconlon @DanielleThierryUSDSVA the idea here was to create a page that was **only** in staging which would have a `DIV ID` to mount a `react app` for the HRT Landing page, which allows us to create a static page in staging with no fear of it being released to production.  If the page is released via Drupal (with a content block) with react app it will get pushed to production which is not the desired approach.  We did this for the VA.gov/contact-us/ page when testing a few links and it worked really well.

**Comment by davidconlon** [comment]
We're double and triple handling this page at this point.
We're using the drupal version of the page with our stakeholders to finalize the content, and @DanielleThierryUSDSVA has lead on that. We should consider creating the static version on staging blocked until we finalize content otherwise it's death by 1000 cuts.

Once the content has been finalized, then we can revisit the idea of creating a page in staging, but I know that @tmac9494 just had some pretty big issues here, so we'll want to check through them to be sure that we can still deliver on that stuff.

**Comment by cindymerrill** [comment]
@davidconlon @brianalloyd Am I correct in understanding from Dave's comments above that the content on the static landing page is currently changing a lot, and so it's too early to make a copy on Staging?

**Comment by brianalloyd** [comment]
@davidconlon thanks for the clarification, we definitely made the assumption that the content had been approved (outside of the numbers stuff), we'll move this to blocked as it would definitely optimize actions if we can get a final content copy before doing FE magic for staging copy.

**Comment by cindymerrill** [comment]
Today reached out to Danielle Thierry and Martha Wilkes over Slack about content revisions needed (at least phone numbers) for the HRT landing page and poster.

**Comment by cindymerrill** [comment]
@brianalloyd I just reviewed the [landing page preview](http://preview-prod.vfs.va.gov/preview?nodeId=34641) from Danielle, and it looks great to me. There have been a LOT of changes since @trent put the last version up on Staging, so he'll need to grab the content blocks all over again.

Here's the feedback I had on Trent's last version on Staging, which I copied from slack [here](https://dsva.slack.com/archives/C52CL1PKQ/p1643917931308419):

1. Remove “Harassment Report Tool” from the top header and breadcrumb.
2. Under “On this page”, the 5 jump links are each displayed twice and shouldn’t be. They'll need to be updated to the new version, but each jump link should be displayed only once, not twice.
3. The two “[Learn more about filing a report with the OIG](https://staging.va.gov/va.gov/oig/hotline/)” links lead to a page that doesn’t exist
4. The header “Report harassment as a Veteran or visitor to VA” and text below aren’t correctly responsive. Making the display size wider results in “Veteran or visitor to VA” to appear on the second line, but making it narrower results in only “or visitor to VA” on the second line and so more text on the first line (see 2 screenshots below with black outline indicating the actual window size)
![image](https://user-images.githubusercontent.com/69914583/155589695-c71c98cb-6444-47b9-8095-a49334e27834.png)

Let me know if you have any questions, @tmac9494. I’m happy to discuss any of it with you. When do you anticipate having it ready for me to review? **I'd like to see it as early as possible next week so we have time to fix any issues before the end of the week** in preparation for pilot testing on Monday Mar 7. Thanks!

**Comment by tmac9494** [comment]
https://github.com/department-of-veterans-affairs/vagov-content/pull/824

**Comment by cindymerrill** [comment]
When I finally got to see the landing page on Staging today, I found that the content is out-of-date and doesn't reflect the significantly revised version that Danielle shared last week. Trent said he was unable to access that version and asked for the source code for the page, which I sent him. Hopefully now he can now build it on Staging. I would like to review it on Friday before pilot testing on Monday and Tuesday.

**Comment by cindymerrill** [comment]
This morning Trent asked me for another copy of the HTML for the revised landing page, which I provided. I still haven't seen the revised page up on Staging, and so I'm very concerned about my pilot test on Monday. @brianalloyd

**Comment by cindymerrill** [comment]
Very quickly reviewed on Mon Mar 7 and found the following issues with the landing page on Staging:

1. The text that belongs in the top alert box is below it
2. The first jump link shouldn't exist (because the text block should be inside the alert box), and the last jump link doesn't work
3. Delete code at the bottom of the page just above "More helpful information", which might be why the jump-link isn't working correctly 
4. Missing spaces in many places:
	- missing in "cancome" 
	- missing in "youand" and "otherswho"
	- missing in "foryour", "informationso", "canconduct", "youwhen"
	- missing a space after many boldface periods and commas
	- Most phone numbers and text numbers are missing a space right before and after, making them hard to read
	- Many colons don't have a space after them

@tmac9494 @longmd @brianalloyd

**Comment by cindymerrill** [comment]
Waiting to review to see that the above changes have been made @tmac9494 @longmd

**Comment by brianalloyd** [comment]
Page has been created, however final updates remain InProgress.  Closing this ticket out and will open a separate ticket for any remaining updates.

**Comment by cindymerrill** [comment]
@DanielleThierryUSDSVA has a few updated needed for the landing page @brianalloyd @tmac9494

**Comment by brianalloyd** [comment]
@DanielleThierryUSDSVA I created a follow on ticket #38202 if you could provide the needed updates in that ticket for action.  Let's plan a sync on Thursday if possible so we can review and validate in support of next weeks research sessions.
